### PR TITLE
Add license CAL-1.0

### PR DESCRIPTION
**Pull request by goneall**
License text was copied form the OSI web page.

License name was changed from:
"Cryptographic Autonomy License version 1.0"
to:
"Cryptographic Autonomy License Version 1.0"
based on an error reported in compile.py.

Signed-off-by: Gary O'Neall <gary@sourceauditor.com>

### CODE REVIEW

**Comment by goneall** [comment]
This license is listed in issue #62

File: texts/plain/CAL-1.0 [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+or
+
+SPDX-License-Identifier: CAL-1.0-Combined-Work-Exception
+Licensed under the Cryptographic Autonomy License version 1.0, with Combined Work Exceptio
+
+______________________________________________________________________
+
+1. Purpose
+This License gives You unlimited permission to use and modify the software to which it applies (the �Work�), either as-is or in modified form, for Your private purposes, while protecting the owners and contributors to the software from liability.
```
Comment (smaffulli) [inline texts/plain/CAL-1.0:16]: the artifacts here make me think there's an issue with the encoding of the file. Am I wrong?

File: texts/plain/CAL-1.0 [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+This Cryptographic Autonomy License (the �License�) applies to any Work whose owner has marked it with any of the following notices, or a similar demonstration of intent:
+
+SPDX-License-Identifier: CAL-1.0
+Licensed under the Cryptographic Autonomy License version 1.0
+
+or
+
+SPDX-License-Identifier: CAL-1.0-Combined-Work-Exception
+Licensed under the Cryptographic Autonomy License version 1.0, with Combined Work Exceptio
```
Comment (smaffulli) [inline texts/plain/CAL-1.0:11]: this looks like a typo. It's also present on https://opensource.org/licenses/CAL-1.0 but not on https://spdx.org/licenses/CAL-1.0.html

**Review by smaffulli — CHANGES_REQUESTED** [review]
I think there are issues with backticks in the text and a typo (which I just fixed in the OSI website)

**Comment by webmink** [comment]
@goneall Change request and merge conflict here

File: texts/plain/CAL-1.0 (hunk 1 above)
Comment (goneall) [reply texts/plain/CAL-1.0]: Fixed the typo in the PR and changed the encoding to UTF-8 which should resolve the issues.

File: texts/plain/CAL-1.0 (hunk 2 above)
Comment (goneall) [reply texts/plain/CAL-1.0]: Resolved in the latest PR

**Comment by goneall** [comment]
@webmink this PR should be good to go.  Resolved merge conflict and made changes per the review comments.

**Pull request merged**